libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthentication.cs

```diff
@@ -59,6 +59,26 @@ public virtual BotFrameworkClient CreateBotFrameworkClient()
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Gets the originating audience from Bot OAuth scope.
+        /// </summary>
+        /// <returns>The originating audience.</returns>
+        public virtual string GetOriginatingAudience()
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Validate Bot Framework Protocol requests.
+        /// </summary>
+        /// <param name="authHeader">The http auth header.</param>
+        /// <param name="cancellationToken">A cancellation token.</param>
+        /// <returns>Asynchronous Task with <see cref="ClaimsIdentity"/>.</returns>
+        public virtual Task<ClaimsIdentity> ValidateSkillsAuthHeaderAsync(string authHeader, CancellationToken cancellationToken)
```

Comment:
The other methods in the class start with AuthenticateXyz, consider renaming it.
```suggestion
        public virtual Task<ClaimsIdentity> AuthenticateSkillRequestAsync(string authHeader, CancellationToken cancellationToken)
```

Reply:
Renamed to `AuthenticateChannelRequestAsync`